crates/forge/bin/cmd/fmt.rs

```diff
@@ -149,7 +149,7 @@ impl FmtArgs {
             Input::Stdin(source) => format(source, None).map(|diff| vec![diff]),
             Input::Paths(paths) => {
                 if paths.is_empty() {
-                    cli_warn!(
+                    sh_warn!(
```

Comment:
I believe there are some manual "warn"/"warning"/yellow formatting, we should look into replacing these as well

Reply:
will be handled as part of larger search + replace